inc/classes/class_auth.php

```diff
@@ -52,9 +52,18 @@ public function __construct($frmwrkmode = "")
 
         // Better handle it here, otherwise its an DB-Query for each $dsp->FetchUserIcon()
         $res = $db->qry(
-            'SELECT userid,lasthit FROM %prefix%stats_auth
-            WHERE login = "1" AND (lasthit > %int% OR lastajaxhit > %int%) AND userid > 0
-            GROUP BY userid',
+            'SELECT
+                userid,
+                lasthit
+            FROM %prefix%stats_auth
+            WHERE
+                login = "1"
+                AND (
+                    lasthit > %int%
+                    OR lastajaxhit > %int%
+                )
+                AND userid > 0
+            GROUP BY userid, lasthit',
```

Comment:
I suspect this does not return the same user count if a user is logged in in two separate sessions. In that case  both fields in the GROUP BY clause are unique (unless last access was done in the same second in both sessions) returning two results instead of one as previously

Reply:
I will re-check this in the next days and will find a good solution for this.

Reply:
I guess either a direct selection by the MySQL time functions (something like `SELECT DISTINCT(userid) WHERE TIMESTAMPDIFF(MINUTE,lasthit,NOW())<10 AND TIMESTAMPDIFF(MINUTE,lastajaxhit,NOW())<1)`(so that lasthit is not needed in the later code) or a direct counting in MySQL and removal of the later PHP code should do the trick. 
But haven't tested it and this by far from being a critical issue, of course.
I would rather suggest to raise a ticket and spend time on it whenever we got really nothing better to do ;)

Reply:
I fixed the regression in https://github.com/lansuite/lansuite/pull/139